### DISCUSSION TITLE
No Listing for Sequent 8 Relay Hat V4... ?

### DISCUSSION THREAD

**Discussion by rock-crusher** [tagyoureit/nodejs-poolController#574] [REM] [answered]
Just purchased another Sequent 8 relay hat in order to have more expansion for yard lighting etc. I have two now, the first was purchased in 2021 and is a version 4, you know, the one with the printing backwards for relay indication (annoying). The new one is a version 5 and this time they got the printing correct for relay indication. V4 has green LED indicators and V5 has red LED. Anyway, when connecting to REM the control drop down menu list includes V2, V3, & V5 for the Sequent 8 relay hat, but for some reason it skips version 4 (V4), what happened? What version should I pick for proper operation of V4, or doesn’t it really matter.  
Maybe it could be added to the menu?

**Comment by rstrouse** [comment, marked as answer]
Actually, the versioning thing came much later.  Version 4 used to be called the Sequent 8 IND and the V2 and V3 versions were actually called Sequent 8 Relay and Sequent Relay 8.  Yeah no kidding.   I think there was a bit of confusion because the relay mapping was different between all of them until v5.0 used the same mapping as the IND.  Use (IND) v5.0 for both.

I believe the primary differences between them are the screen printing and perhaps some supporting chips since that I know they had to source other suppliers.  I sent them an e-mail when the versions started changing and got the skinny that they never intended to change the underlying relay mapping or command registers.  The problem boiled down to limitations in finding chips so they had to rework the designs or have no product.  Again Relay 8 IND (v5) should be used for both.

To add to this confusion if you try to find a source repo for v4 on the Sequent site you will be disappointed as the links go nowhere.

**Reply by rock-crusher** [reply]
Thanks again rstrouse, you are a wealth of information and as always I appreciate your help.
